### PR TITLE
clarified service account type

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6969

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-dir-sync.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Sign-in a domain controller or management workstation with _Domain Admin_ equiva
 2. Click the **Users** container in the navigation pane.
 3. Right-click **Key Admins** in the details pane and click **Properties**.
 4. Click the **Members** tab and click **Add**
-5. In the **Enter the object names to select** text box, type the name of the Azure AD Connect service account.  Click **OK**.
+5. In the **Enter the object names to select** text box, type the name of the service account used as an AD DS Connector account.  Click **OK**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-dir-sync.md:42]:
```suggestion
5. In the **Enter the object names to select** text box, type the name of the service account used as an AD DS Connector account and click **OK**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nam31** [comment]
@mapalko  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by mapalko — APPROVED** [review]
I approve of the change but I do not have write access. Someone else will need to merge the PR.

**Pull request merged**